lib/ain-evm/src/precompiles/mod.rs

```diff
@@ -84,17 +84,11 @@ pub struct MetachainPrecompiles;
 // Ref: Ethereum Yellow Paper (https://ethereum.github.io/yellowpaper/paper.pdf) Page 12
 impl MetachainPrecompiles {
     pub fn used_addresses() -> [H160; 9] {
-        [
-            hash(1),
-            hash(2),
-            hash(3),
-            hash(4),
-            hash(5),
-            hash(6),
-            hash(7),
-            hash(8),
-            hash(9),
-        ]
+        (1..=1024)
```

Comment:
not clear what's the rational behind of setting `1024`.. but the return should follow the same??
(1..=1024) -> [H160; 1024]
or
(1..=9) -> [H160; 9]

Reply:
We are reserving additional addresses for future use.